.github/workflows/openapi-compatibility-tests.yaml

```diff
@@ -0,0 +1,40 @@
+name: lakeFS OpenAPI Compatibility Tests
+
+on:
+  pull_request:
+    paths:
+      - "api/**"
+  push:
+    branches:
+      - master
```

Comment:
can be removed until we have a stable version - this will check the master with master

Reply:
why?
Wouldn't it check master with the current commit?